var/spack/repos/builtin/packages/py-nbdime/package.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyNbdime(PythonPackage):
+    """Diff and merge of Jupyter Notebooks"""
+
+    homepage = "https://nbdime.readthedocs.io/"
+    pypi     = "nbdime/nbdime-3.1.1.tar.gz"
+
+    version('3.1.1', sha256='67767320e971374f701a175aa59abd3a554723039d39fae908e72d16330d648b')
+
+    depends_on('python@3.6:',        type=('build', 'run'))
+    depends_on('py-setuptools@40.8.0:',            type='build')
+    depends_on('py-nbformat',        type=('build', 'run'))
+    depends_on('py-colorama',        type=('build', 'run'))
+    depends_on('py-pygments',        type=('build', 'run'))
+    depends_on('py-tornado',         type=('build', 'run'))
+    depends_on('py-requests',        type=('build', 'run'))
+    depends_on('py-gitpython@:2.1.3,2.1.7:',       type=('build', 'run'))
+    depends_on('py-jupyter-server',  type=('build', 'run'))
+    depends_on('py-jupyter-server-mathjax@0.2.2:', type=('build', 'run'))
+    depends_on('py-jinja2@2.9:',     type=('build', 'run'))
+    # From pyproject.toml
+    depends_on('py-jupyterlab@3.0:3', type=('build', 'run'))
```

Comment:
This in `pyproject.toml` are usually build-type only. Do you see any imports that suggest that it's needed at run-time?

Reply:
It has a command to register/deregister itself from Jupyter: https://github.com/jupyter/nbdime/blob/master/nbdime/__main__.py#L44-L54 . Does this count as a reason to have Jupyter at runtime?

Reply:
Works for me